content/pt-br/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -0,0 +1,125 @@
+
+O Kubespray oferece a habilidade de customizar vários aspectos do deployment:
+
+* Escolha do modo de deployment: kubeadm ou non-kubeadm
+* Plugins CNI (rede)
+* Configuração de DNS
+* Escolha da camada de gerenciamento: nativo/binário ou conteinerizado
+* Versões de componentes
+* Reflexões de rota Calico
+* Opções de agente de execução do componente
+  * {{< glossary_tooltip term_id="docker" >}}
+  * {{< glossary_tooltip term_id="containerd" >}}
+  * {{< glossary_tooltip term_id="cri-o" >}}
+* Métodos de geração de certificados
+
+As customizações do Kubespray podem ser feitas em um [arquivo variável](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html). Se você estiver começando com o Kubespray, considere usar os padrões do Kubespray para implantar seu cluster e explorar o Kubernetes.
+
+### (4/5) Deploy um Cluster
+
+Em seguida, deploy seu cluster:
+
+Deployment do cluster usando [ansible-playbook](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#starting-custom-deployment).
+
+```shell
+ansible-playbook -i your/inventory/inventory.ini cluster.yml -b -v \
+  --private-key=~/.ssh/private_key
+```
+
+Grandes deployments (mais de 100 nós) podem exigir [adjustes específicos](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/large-deployments.md) para obter melhores resultados.
+
+### (5/5) Verifique o deployment
+
+O Kubespray fornece uma maneira de verificar a conectividade entre pods e a resolução de DNS com o [Netchecker](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/netcheck.md). O Netchecker garante que os pods de agentes do netchecker possam resolver solicitações de DNS e pingar cada um dentro do namespace padrão. Esses pods imitam um comportamento semelhante ao restante das cargas de trabalho e servem como indicadores de integridade do cluster.
+
+## Operações do Cluster
+
+O Kubespray fornece playbooks adicionais para gerenciar seu cluster: _dimensionar_ e _atualizar_.
+
+### Dimensione seu cluster
+
+Você pode adicionar nós de processamento de seu cluster executando o playbook de escala. Para obter mais informações, consulte "[Adicionando nós](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#adding-nodes)".
+Você pode remover nós de processamento de seu cluster executando o playbook remove-nó. Para obter mais informações, consulte "[Remover nós](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#remove-nodes)".
+
+### Atualize seu cluster
+
+Você pode atualizar seu cluster executando o playbook de atualização do cluster. Para obter mais informações, consulte "[Atualizações](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/upgrades.md)".
+
+## Limpeza
+
+Você pode redefinir seus nós e eliminar todos os componentes instalados com o Kubespray por meio do playbook de [redefinição](https://github.com/kubernetes-sigs/kubespray/blob/master/reset.yml).
+
+{{< Cuidado >}}
```

Comment:
@juliafmorgado this needs to be changed too: {{< Cuidado >}} -> {{< /caution >}}
With this the build error will be fixed :D